homeassistant/components/plex/media_player.py

```diff
@@ -98,6 +109,13 @@ def setup_platform(hass, config, add_entities_callback, discovery_info=None):
         token = None
         has_ssl = False
         verify_ssl = True
+    elif config:
+        host = config.get(CONF_HOST)
+        port = config.get(CONF_PORT)
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.